test/pylib/scylla_cluster.py

```diff
@@ -1249,6 +1278,21 @@ async def _after_test(self, _request) -> str:
             self.current_test_case_full_name = ''
         self.is_after_test_ok = True
         cluster_str = str(self.cluster)
+
+        # cancel all other tasks in ScyllaClusterManager
+        self.logger.info(self.repr_tasks_history())
+        for key, value in self.tasks_history.items():
+            if "after-test" not in value.path_qs:
```

Comment:
I think the implementation of `_after_test` should not depend on how the route that calls `_after_test` is named -- it's an abstraction leak, dependency reversal.

Reply:
logic changed, now I'm looking into task name